flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/SuccessfulCheckpoint.java

```diff
@@ -19,29 +19,28 @@
 package org.apache.flink.runtime.checkpoint;
 
 import org.apache.flink.api.common.JobID;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
+import java.io.Serializable;
+import java.util.ArrayList;
 import java.util.List;
 
 /**
  * A successful checkpoint describes a checkpoint after all required tasks acknowledged it (with their state)
  * and that is considered completed.
  */
-public class SuccessfulCheckpoint {
-	
-	private static final Logger LOG = LoggerFactory.getLogger(SuccessfulCheckpoint.class);
-	
+public class SuccessfulCheckpoint implements Serializable {
```

Comment:
Maybe we can rename this class to `CompletedCheckpoint`, because there are methods like `PendingCheckpoint.toCompletedCheckpoint()` which indicate that you get a `CompletedCheckpoint` instead of a `SuccessfulCheckpoint`.